app/models/tag.rb

```diff
@@ -87,7 +87,7 @@ def calculate_hotness_score
     self.hotness_score = Article.tagged_with(name).
       where("articles.featured_number > ?", 7.days.ago.to_i).
       map do |article|
-        (article.comments_count * 14) + (article.reactions_count * 4) + rand(6) + ((taggings_count + 1) / 2)
+        (article.comments_count * 14) + (article.positive_reactions_count * 4) + rand(6) + ((taggings_count + 1) / 2)
```

Comment:
Just thinking out loud here. I think the multiplier could be reduced a bit as reaction count fluctuate more than comment count. Looks like we're calculating based on articles featured in the recent week so it's not going to be a gigantic change. Maybe we could simplify this by not giving any weights (ie `article.comments_count + article.positive_reactions_count`)? What's your thought @benhalpern ?

A bit off topic but I'm wondering how might these affect the formula too.
- What is the purpose of `rand(6)`? a dice roll?
- Why `taggings_count + 1` ?